publication-commons/src/main/java/no/unit/nva/publication/permissions/file/FileStrategyBase.java

```diff
@@ -11,9 +14,34 @@ public class FileStrategyBase {
 
     protected final FileEntry file;
     protected final UserInstance userInstance;
+    protected final Resource resource;
 
-    protected FileStrategyBase(FileEntry file, UserInstance userInstance) {
+    protected FileStrategyBase(FileEntry file, UserInstance userInstance, Resource resource) {
         this.file = file;
         this.userInstance = userInstance;
+        this.resource = resource;
+    }
+
+    protected boolean hasAccessRight(AccessRight accessRight) {
+        return nonNull(userInstance) && userInstance.getAccessRights().contains(accessRight);
+    }
+
+    protected boolean currentUserIsFileCuratorForGivenFile() {
+        if (!currentUserIsFileCurator()) {
```

Comment:
Is this only ever called with negation? Move the negation to the method?

Reply:
but I don't like it negation in method names! :-P  found a way to avoid it